src/containers/board/BoardContainer.jsx

```diff
@@ -13,13 +14,19 @@ class BoardContainer extends Component {
     this.props.cleanListsState()
     this.props.cleanCardsState()
     this.props.getAllListsInBoard(this.props.match.params.boardId)
-    this.props.getOneBoard(this.props.match.params.boardId)    
+    this.props.getOneBoard(this.props.match.params.boardId)
   }
 
   render() {
-    return (
-      <DndBoard {...this.props} {...this.props.board} boardId={this.props.match.params.boardId} />
-    )
+    if (this.props.board.isFailure) {
```

Comment:
Simplify the syntax: 

```jsx
return 
  { this.props.board.isFailure ? (
    <ErrorDisplayer message="Failed to load this board. Maybe it doesn't exist, or you don't have access to it." />
  ) : (
    <DndBoard {...this.props} {...this.props.board} boardId={this.props.match.params.boardId} />
  )}
```

Reply:
compilation error (unreachable block + break the app)